Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rails:4.2.6
```

Comment:
I would consider using the `ruby` image as a base instead of `rails`. Either that or put some sort of check in below to ensure that the version used here matches your `Gemfile.lock`